### PR TITLE
Add Neuledge

### PR DESCRIPTION
**Pull request by moshest**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ For the latest additions [click here](https://github.com/agamm/awesome-developer
 * [Supabase](https://supabase.io/) - Hosted and open source managed postgres DB like firebase.
 * [upstash](https://upstash.com/) - Serverless tailored redis provider.
 * [Apitable](https://apitable.com/) - API-oritented Airtable alternative.
+* [Neuledge](https://neuledge.com/) - Universal language for databases.
```
Comment (agamm) [inline README.md:118]:
```suggestion
* [Neuledge](https://neuledge.com/) - Universal database agnostic language (abstract ORM).
```

**Review by agamm — CHANGES_REQUESTED** [review]
What about this?

**Comment by moshest** [comment]
Sound good

**Review by agamm — APPROVED** [review]
LGTM!

**Comment by agamm** [comment]
If you're at it we need to sort the list too from the changes...

**Comment by moshest** [comment]
Done.. Please approve

**Review by agamm — APPROVED** [review]
Thanks again :)

**Comment by fmerian** [comment]
welcome to the list, @moshest :)

**Pull request merged**